checker/src/main/java/org/checkerframework/checker/i18nformatter/I18nFormatterVisitor.java

```diff
@@ -87,9 +90,14 @@ private void checkInvocationFormatFor(I18nFormatCall fc) {
                                     break;
                                 default:
                                     if (!fc.isValidParameter(formatCat, paramType)) {
+                                        ExecutableElement method =
```

Comment:
I would probably move this outside of the loop, to make clear that it doesn't change each iteration.

Reply:
Same reasoning as for similar case above.